docker-compose.yaml

```diff
@@ -37,6 +37,7 @@ services:
       - ${NODE_PORT}:8000
     volumes:
       - /etc/localtime:/etc/localtime:ro
+      - .:/app
```

Comment:
This is only for testing, right? Because the app application should be used as a Docker image (line 34)